server/src/main/java/org/apache/druid/server/coordinator/duty/KillUnusedSegments.java

```diff
@@ -149,7 +173,7 @@ private void killUnusedSegments(Collection<String> dataSourcesToKill)
       }
     }
 
-    log.debug("Submitted kill tasks for [%d] datasources.", submittedTasks);
+    log.debug("Submitted [%d] kill tasks for [%d] datasources.", submittedTasks, dataSourcesToKill.size());
```

Comment:
if any datasources were skipped due to not enough slots, we could consider logging them here?

Reply:
Will get this in a follow up patch